TIL/topics/dart_basics/VariousDataFormats.md

```
@@ -0,0 +1,120 @@
+## VariousDataFormats(여러가지 데이터 형식)
+---
+### 다양한 가공된 데이터의 형식을 다룬다.
+- CSV
+	- ','를 사용하여 데이터를 나누는 형식
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)